controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -655,8 +655,7 @@ bool master_thread::handle_cmdu_1905_autoconfiguration_WSC(Socket *sd,
         }
     }
 
-    if (database.setting_certification_mode() ||
```

Comment:
It should actually have been
```
    if (!database.setting_certification_mode() &&
```

Reply:
We have to handle VS messages also in certification mode, for the local agent